zbsprimarystorage/zbsprimarystorage/zbsutils.py

```
@@ -0,0 +1,133 @@
+__author__ = 'Xingwei Yu'
+
+import os
+
+import zstacklib.utils.jsonobject as jsonobject
+
+from zstacklib.utils.bash import *
```

Ruff (0.8.2)

`from zstacklib.utils.bash import *` used; unable to detect undefined names

(F403)